examples/monitors-signals/main.tf

```
@@ -55,3 +55,59 @@ resource "chronosphere_monitor" "monitor_with_signal" {
     }
   }
 }
+
+resource "chronosphere_monitor" "monitor_with_logging_query" {
+  name      = "Monitor With Logging Query"
+  bucket_id = chronosphere_bucket.b.id
+  query {
+    logging_expr = <<-EOF
+      service = "nginx" | make-series by chronosphere_namespace
+    EOF
+  }
+  signal_grouping {
+    label_names = ["kubernetes_namespace"]
+  }
+  series_conditions {
+    condition {
+      severity = "warn"
+      value    = 20
+      op       = "GT"
+    }
+
+    override {
+      label_matcher {
+        name  = "app"
+        type  = "EXACT_MATCHER_TYPE"
+        value = "dbmon"
+      }
+
+      condition {
+        severity        = "critical"
+        value           = 1.0
+        op              = "GT"
+        sustain         = "60s"
+        resolve_sustain = "30s"
+      }
+    }
+  }
+
+  labels = {
+    "team" = "my team"
+  }
+  annotations = {
+    "runbook" = "go/runbook"
+  }
+
+  schedule {
+    timezone = "UTC"
+
+    dynamic "range" {
+      for_each = toset(["Monday", "Tuesday", "Wednesday", "Thursday", "Friday"])
+      content {
+        day   = range.key
+        start = "07:00"
+        end   = "20:00"
+      }
+    }
+  }
```

strip down the monitor to a minimal config since goal is to test logging queries

```suggestion
  series_conditions {
    condition {
      severity = "warn"
      value    = 20
      op       = "GT"
    }
  }
```